jasmin.yml

```diff
@@ -7,6 +7,19 @@
     - role: jasmin.cluster-infra
       cluster_network: "caastest-U-internal"
 
+- hosts: openstack
+  tasks:
+    - block:
+        - name: Get stack outputs
+          os_stack_outputs:
+            stack_id: "{{ storage_name }}"
+        - name: Export storage information from stack outputs
+          set_fact:
+            storage_stack_info: "{{ stack_outputs['info'] | select('ne', '') | map('from_json') | selectattr('storage_type', '==', 'nfs') | first }}"
```

Comment:
It would be good in a follow-on development to extract the storage server hostnames so that they can be added to `/etc/hosts` along with the nodes in the Slurm cluster.

Building on this it would also be good to parameterise the specification of NFS.